hugo/content/_index.md

```diff
@@ -6,7 +6,3 @@ css_container: container-fluid
 
 <!-- To disable printing a date for the MOTD, set the "date" field above to some
 value before 2019. -->
-
```

Comment:
This results in an empty blue box now (can be seen on the preview's front page).

You might want to comment out these lines in the template if we don't want to use this box for anything right now: https://github.com/acl-org/acl-anthology/blob/b7d4380974fceda2deff4af08b0f13eaad93f565/hugo/layouts/index.html#L37-L43